model.py

```diff
@@ -307,21 +288,18 @@ def _set_var(self, var, ndarray):
 
         t.set(ndarray, place)
 
-    def _run(self, inputs, labels=None, device='CPU', device_ids=None):
+    def _run(self, inputs, labels=None, device='CPU'):
```

Comment:
The argument `device` seems unused here. Can we remove it

Reply:
Done, Thanks!